### PR TITLE
Feature: Union schema compatibility

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
Issue: #55 

Confirm the following files were correctly updated automatically:
- [x] CHANGELOG
- [x] README
- [x] stg_google_ads.yml, src_google_ads.yml, google_ads.yml (depending if source or transform)
- [x] docs.md
- [x] joins
- [x] window functions (partition by)
- [x] 'source_relation' column added in staging and transform models
- [x] tests

Manual updates:
- [x] Add `source_relation` to downstream models if necessary
- [x] Finish any incomplete/incorrect joins/partitions
- [x] Update tests to include `source_relation` in unique-combination-of-cols if necessary
    - May need to remove some uniqueness tests in favor of the combo test
    
Validation:
- [x] `dbt run` locally passes
- [x] `dbt test` locally passes
- See internal ticket for the validation that tests multiple sources

### CODE REVIEW

**Review by fivetran-joemarkiewicz — APPROVED** [review]
These changes look good to me! I have the one outstanding request for the CHANGELOG and README update. Once that is applied, this will be good to go after the others are approved.

File: CHANGELOG.md
Comment (fivetran-catfritz) [reply CHANGELOG.md]: Updated.

**Pull request merged**
